src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java

```diff
@@ -60,7 +63,14 @@ public CatalogService() {
         }
         return services;
     }
-    
+
+    public String getServiceCatalog(String member, String service) throws FileNotFoundException {
```

Comment:
this is a place holder. it's gonna be replaced by the proper integration with cache.